rspace/src/test/scala/coop/rchain/rspace/nextgenrspace/ReplayRSpaceTests.scala

```diff
@@ -789,6 +790,501 @@ trait ReplayRSpaceTests extends ReplayRSpaceTestsBase[String, Pattern, String, S
       }
   }
 
+  "Picking a datum from 100 waiting datums" should "replay correctly" in fixture {
```

Comment:
I'm failing to see the value of having this test along with "Picking n datums from m waiting datums"

Reply:
This whole commit looks unnecessary